tools/azure-devtools/src/azure_devtools/scenario_tests/base.py

```diff
@@ -112,6 +113,11 @@ def __init__(self,  # pylint: disable=too-many-arguments
             filter_headers=self.FILTER_HEADERS
         )
         self.vcr.register_matcher('query', self._custom_request_query_matcher)
+        if match_body:
```

Comment:
I feel I'd prefer to have it as "match_on", just in case we need more complicated later, I don't want to add match_port, match_raw_body, etc.
https://vcrpy.readthedocs.io/en/latest/configuration.html#request-matching

Reply:
Discussed offline, approving